src/routes/index.svelte

```diff
@@ -168,6 +169,18 @@
 	};
 
 	onMount(() => {
+		const darkMode = () => {
+			const dark: boolean = window.matchMedia('(prefers-color-scheme: dark').matches;
```

Comment:
typo: should be `.matchMedia('(prefers-color-scheme: dark)')`

Reply:
Cat might have been on top of my keyboard. Sorry, fixing.